pkg/storage/storage.go

```diff
@@ -35,7 +35,7 @@ func NewPaginationOptions(ps int32, contToken string) PaginationOptions {
 
 // Writes and Deletes are typesafe aliases for Write arguments.
 type Writes = []*openfgav1.TupleKey
-type Deletes = []*openfgav1.TupleKey
```

Comment:
This update highlights that our storage interface is a bit to coupled with our protobufs. Ideally we'd take some time to refactor our storage interface so that it’s more agnostic of the protobuf layer. We can make better use of interfaces and abstract storage filter structs there to avoid protobuf impacts.